firmware/library/L1_Drivers/onboardled.hpp

```diff
@@ -0,0 +1,31 @@
+// This provides the functions to manipulate the numbered LEDs that are soldered
+// onto the SJTwo Board.
```

Comment:
Remove the reference to the SJTwo. This whole framework, may become cross platform in the future.